weblate/vcs/git.py

```diff
@@ -574,7 +602,7 @@ def fork(self):
         """Create fork of original repository if one doesn't exist yet."""
         remotes = self.execute(["remote"]).splitlines()
         if self.get_username() not in remotes:
-            self.execute(["fork"])
+            super().fork()
```

Comment:
This looks wrong, the super class (GitRepository) should have no knowledge on forking, it's the child classes (GithubRepository, GitLabRepository, ..) which are supposed to implement the actual logic.

Reply:
I will move it to the child classes. Makes more sense.